src/lib/isMobilePhone.js

```diff
@@ -66,6 +66,7 @@ const phones = {
   'lt-LT': /^(\+370|8)\d{8}$/,
   'ms-MY': /^(\+?6?01){1}(([0145]{1}(\-|\s)?\d{7,8})|([236789]{1}(\s|\-)?\d{7}))$/,
   'nb-NO': /^(\+?47)?[49]\d{7}$/,
+  'ne-NP': /^(\+?977)?9[78]\d{8}$/,
```

Comment:
Good work here, could you add `some tests` and update the section of `isMobilePhone` in the `README`. 
In case you need any support or help, please let me know or leave a comment and you will get feedback.

Reply:
sure